api/v1alpha2/virtualmachine_network_types.go

```diff
@@ -19,7 +19,9 @@ type VirtualMachineNetworkRouteSpec struct {
 	Via string `json:"via"`
 
 	// Metric is the weight/priority of the route.
-	Metric int32 `json:"metric"`
+	//
+	// +optional
+	Metric int32 `json:"metric,omitempty"`
```

Comment:
Should this not be nillable if we want it to be optional?

Reply:
BV told me offline the netplan docs state this must be a positive int, so we'll add a min value of 1.